frontend/src/components/StationMiniCard/StationMiniCard.jsx

```diff
@@ -0,0 +1,36 @@
+import PropTypes from "prop-types";
+import NavbarStations from "../../assets/navbar_stations.svg";
+import "./StationMiniCard.scss";
+
+function StationMiniCard({ stations }) {
+  return (
+    <article className="station_mc">
```

Comment:
Article or section?

Reply:
Article changed for "li" elements mapped into an "ul"